server/lobbyconnection.py

```diff
@@ -628,6 +633,28 @@ def check_version(self, message):
         self.send_game_list()
         await self.send_tutorial_section()
 
+    def command_restore_game_session(self, message):
+        game_id = message.get('game_id')
+
+        # Restore the player's game connection, if the game still exists and is live
+        if not game_id or game_id not in self.game_service:
+            self.send_warning("The game you were connected to does no longer exist")
+            return
+
+        game = self.game_service[game_id]  # type: Game
+        if game.state != GameState.LOBBY and game.state != GameState.LIVE:
+            self.send_warning("The game you were connected to is no longer available")
+            return
+
+        self._logger.debug("Restoring game session of player %s to game %s", self.player, game)
+        self.game_connection = GameConnection(self.loop, self, self.player_service, self.game_service, self.player,
```

Comment:
Let's use kwargs for GameConnection constructor. Some of these aren't needed anymore. (Also fix indents)